src/scrapers/mizrahi.js

```diff
@@ -0,0 +1,155 @@
+import moment from 'moment';
+import {
+  SHEKEL_CURRENCY,
+  NORMAL_TXN_TYPE,
+  TRANSACTION_STATUS,
+  ISO_DATE_FORMAT,
+} from '../constants';
+import { BaseScraperWithBrowser, LOGIN_RESULT } from './base-scraper-with-browser';
+import { fetchPostWithinPage } from '../helpers/fetch';
+import { waitForNavigation } from '../helpers/navigation';
+import { pageEvalAll } from '../helpers/elements-interactions';
+
+const BASE_URL = 'https://www.mizrahi-tefahot.co.il';
+const LOGIN_URL = `${BASE_URL}/he/bank/Pages/Default.aspx`;
+const AFTER_LOGIN_BASE_URL = /https:\/\/mto\.mizrahi-tefahot\.co\.il\/ngOnline\/index\.html#\/main\/uis/;
```

Comment:
@baruchiro why aren't we using a regular string here? would also be great to reuse `https://mto.mizrahi-tefahot.co.il` as something like `BASE_APP_URL`.

Reply:
I mentioned you in a comment above, but I'm not sure what variable you mean?
I thought you meant `AFTER_LOGIN_BASE_URL`, but maybe you meant `BASE_URL` and/or `LOGIN_URL`?

Reply:
@baruchiro no, I just referred to `https://mto.mizrahi-tefahot.co.il` and adding it to it's own constant (because it's also being used below 👇)

Reply:
@eshaham Solved in https://github.com/eshaham/israeli-bank-scrapers/pull/256/commits/c8ad9fe388f90f45675e4fd3f1056e339e93f841, but I can't use the `BASE_APP_URL` in `AFTER_LOGIN_BASE_URL` because it must to be a `ReGex` object.

Reply:
👍